### PR TITLE
Filesystems test update

### PR DESCRIPTION
**Pull request by jrusz**
This pull request includes:

- [ ] adequate testing for the new functionality or fixed issue
- [ ] adequate documentation informing people about the change such as
  - [ ] submit a PR for the [guides](https://github.com/osbuild/guides) repository if this PR changed any behavior described there: https://www.osbuild.org/guides/

### CODE REVIEW

**Comment by jrusz** [comment]
@atodorov Can you please review the first commit? I'm going to remove the commented code and drop the other two commits if you think it's fine.
Passing pipeline: https://gitlab.com/redhat/services/products/image-builder/ci/osbuild-composer/-/pipelines/591619561
Deliberately failed pipeline: https://gitlab.com/redhat/services/products/image-builder/ci/osbuild-composer/-/pipelines/592120193
I've also checked the second scenario locally and is imo working reliably like this

File: test/cases/filesystem.sh [hunk 1]
```diff
@@ -203,14 +203,22 @@ INFO="$(sudo /usr/libexec/osbuild-composer-test/image-info "${IMAGE_FILENAME}")"
...
 done
 
+if [ ${#FAILED_MOUNTPOINTS[@]} -eq 0 ]; then
+  echo "🎉 Success scenario has passed."
+else
+  echo "🔥 Success scenario has failed. The following mountpoints were not present:"
+  printf '%s\n' "${FAILED_MOUNTPOINTS[@]}"
+  exit 1
+fi
+
```
Comment (atodorov) [inline test/cases/filesystem.sh:217]: I would prefer this to be a function to avoid repetition. Otherwise looks good.

**Review by atodorov — CHANGES_REQUESTED** [review]
Minor changes requested.

**Review by atodorov — DISMISSED** [review]

**Comment by jrusz** [comment]
@atodorov I cherry-picked your commit disabling the `cloud-image-val` because it's now failing on Fedora-35 as well and effectively blocking the CI.

**Comment by atodorov** [comment]
> `cloud-image-val` because it's now failing on Fedora-35

oh boy

**Review by atodorov — APPROVED** [review]

**Pull request merged**